### PR TITLE
External request template

### PR DESCRIPTION
**Pull request by Metfriet**

### CODE REVIEW

File: podium-gateway/src/main/resources/config/application.yml [hunk 1]
```diff
@@ -142,7 +142,7 @@ podium:
     ribbon:
         display-on-active-profiles: dev
     files:
-        upload-dir: ./podium_uploads
+        upload-dir: /home/frank/code/podium/podium_uploads
```
Comment (rnugraha) [inline podium-gateway/src/main/resources/config/application.yml]: is this accidental?

File: podium-gateway/src/test/java/nl/thehyve/podium/web/rest/RequestResourceIntTest.java [hunk 2]
```diff
@@ -711,4 +711,62 @@ public void submitReviewFeedbackForWrongUser() throws Exception {
...
+        collect2.put("biobankID", "bbmri-eric:biobankID:BE_B0383");
+        collect2.put("collectionID", organisationUuid1.toString() );
+
+        ArrayList<Map<String, String>> collections = new ArrayList<>();
+        collections.add(collect1);
+        collections.add(collect2);
+        externalRequestRepresentation.setCollections(collections);
+
+        // Submit ext req
+//        ResultActions externalRequest = mockMvc.perform(
```
Comment (rnugraha) [inline podium-gateway/src/test/java/nl/thehyve/podium/web/rest/RequestResourceIntTest.java]: if not use pls remove

File: podium-gateway/src/main/java/nl/thehyve/podium/domain/ExternalRequestTemplate.java [hunk 3]
```diff
@@ -0,0 +1,57 @@
+package nl.thehyve.podium.domain;
```
Comment (rnugraha) [inline podium-gateway/src/main/java/nl/thehyve/podium/domain/ExternalRequestTemplate.java]: add copyright header

File: podium-gateway/src/main/java/nl/thehyve/podium/service/ExternalRequestTemplateService.java [hunk 4]
```diff
@@ -0,0 +1,68 @@
+package nl.thehyve.podium.service;
```
Comment (rnugraha) [inline podium-gateway/src/main/java/nl/thehyve/podium/service/ExternalRequestTemplateService.java]: add copyright header

File: podium-gateway/src/main/java/nl/thehyve/podium/service/dto/ExternalRequestTemplateRepresentation.java [hunk 5]
```diff
@@ -0,0 +1,23 @@
+package nl.thehyve.podium.service.dto;
```
Comment (rnugraha) [inline podium-gateway/src/main/java/nl/thehyve/podium/service/dto/ExternalRequestTemplateRepresentation.java]: add copyright header

File: podium-gateway/src/main/java/nl/thehyve/podium/service/mapper/ExternalRequestTemplateMapper.java [hunk 6]
```diff
@@ -0,0 +1,31 @@
+package nl.thehyve.podium.service.mapper;
```
Comment (rnugraha) [inline podium-gateway/src/main/java/nl/thehyve/podium/service/mapper/ExternalRequestTemplateMapper.java]: add copyright header

File: podium-gateway/src/main/resources/config/application.yml (hunk 1 above)
Comment (Metfriet) [reply podium-gateway/src/main/resources/config/application.yml]: yes, I will remove it

File: podium-gateway/src/test/java/nl/thehyve/podium/web/rest/RequestResourceIntTest.java (hunk 2 above)
Comment (rnugraha) [inline podium-gateway/src/test/java/nl/thehyve/podium/web/rest/RequestResourceIntTest.java]: Please fix if not work

**Pull request merged**